src/routes/analytics/new/NewFromScratchPage.tsx

```diff
@@ -0,0 +1,18 @@
+import { Container, Title } from "@mantine/core"
+import { CreateQuestion } from "@metabase/embedding-sdk-react"
+import { useAtom } from "jotai"
+import { createQuestionKeyAtom } from "../../../store/create"
+
+export const NewFromScratchPage = () => {
+  const [questionKey] = useAtom(createQuestionKeyAtom)
+
+  return (
+    <Container w="100%">
+      <Title fz="28px" mb="md">
+        Create a question
+      </Title>
+
+      <CreateQuestion key={questionKey} />
```

Comment:
We should reload the CreateQuestion component whenever the "Create Question > From Scratch" button is clicked again on the side navigation.

![CleanShot 2567-09-04 at 02 17 33@2x](https://github.com/user-attachments/assets/c06d3abc-ac6b-4c0d-a9e6-6e6b4aa93297)